### PR TITLE
Add functools32

### PR DESCRIPTION
**Pull request by jakirkham**
A backport package of `functools` with the `lru_cache` to python 2.7.

### CODE REVIEW

**Comment by msarahan** [comment]
Please clean up the comments from conda-skeleton.  Also, since this is specific to Python 2.7, should you pin a version dependency on Python at runtime to <3, so that this is never installed in a Python 3 environment?  Does that even make sense?

**Comment by jakirkham** [comment]
Sorry about that. Cleaned up the comments.

Also, the version has been pinned to be some form of 2.7 as it doesn't support 2.6 or earlier either.

File: functools32/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  fn: functools32-3.2.3-2.tar.gz
+  url: https://pypi.python.org/packages/source/f/functools32/functools32-3.2.3-2.tar.gz
+  md5: 09f24ffd9af9f6cd0f63cb9f4e23d4b2
+
+requirements:
+  build:
+    - python >=2.7,<3
+
+  run:
+    - python >=2.7,<3
```
Comment (ccordoba12) [inline functools32/meta.yaml]: This can simplified by using

    2.7*

Comment (jakirkham) [inline functools32/meta.yaml]: Done.

**Comment by ccordoba12** [comment]
It looks good to me. Merging.

**Comment by jakirkham** [comment]
Thanks.

**Pull request merged**